functions/__fzf_search_shell_variables.fish

```diff
@@ -1,19 +1,19 @@
 function __fzf_search_shell_variables --description "Search and inspect shell variables using fzf. Insert the selected variable into the commandline at the cursor."
```

Comment:
Let's declare the arguments and also set up a check for when there are no variables (for people who customized their key bindings) and do something like `echo "The signature of __fzf_search_shell_variables was changed in https://github.com/PatrickF1/fzf.fish/pull/71. Please see the latest conf.d/fzf.fish and update your key bindings."`

Reply:
@PatrickF1 With the newest change, `__fzf_display_value_or_error` becomes unnecessary. Is it fine to remove it (again)?